docs/setup/quick-start.md

```diff
@@ -1,6 +1,9 @@
-There are some ways to set up Sedna, depends on your use case:
+There are some ways to setup Sedna, depends on youruse case:
 - If you have none Kubernetes environment and don't want to install Kubernetes manually, you can follow [the instruction](./all-in-one.md) to install all-in-one Sedna environment.
-- Else you can follow [this instruction](./install.md) to install Sedna on existing Kubernetes cluster.
```

Comment:
While the "Else" here is ambiguous: does it mean "you have K8s" or "want to install K8s manally"? 

Clarification is suggested:
"Otherwise, when Kubernetes is ready, you can follow [this instruction](./install.md) to install Sedna on existing Kubernetes cluster."